### PR TITLE
Feature/save items

### PR DESCRIPTION
**Pull request by TNT05**
Implemented the feature to be able to save any job postings for later use

### CODE REVIEW

File: linkedin-clone/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "web-vitals": "^1.0.1"
   },
   "scripts": {
-    "start": "react-scripts --openssl-legacy-provider start",
+    "start": "react-scripts start",
```
Comment (OussamaJmili) [inline linkedin-clone/package.json:36]: is it working without the legacy?

**Comment by drbladz** [comment]
Nice way to save and display saved posts

**Review by drbladz — APPROVED** [review]
Looks good.

**Review by OussamaJmili — APPROVED** [review]
looks good

File: linkedin-clone/package.json (hunk 1 above)
Comment (drbladz) [reply linkedin-clone/package.json:36]: yes @TNT05 you should add this line back.

Comment (drbladz) [reply linkedin-clone/package.json:36]: it's fine, I will update it in the next PR if necessary

**Pull request merged**